src/Application.php

```diff
@@ -172,7 +172,7 @@ public function run()
         $loop = EventLoopFactory::create();
         $socket = new SocketServer($this->host . ':' . $this->port, $loop);
         $filesystem = Filesystem::create($loop);
-        $requestHandler = new RequestHandler();
+        $requestHandler = new RequestHandler($loop);
```

Comment:
I would pass the loop as a dependency, but as a method parameter.
That makes having a Loop instance required, even if you want to work in a sync way.

Reply:
Yes, I agree 👍